tests/statustest.php

```diff
@@ -20,7 +20,6 @@ class Test_Files_Antivirus_Status extends  \OCA\Files_Antivirus\Tests\Testbase {
 
 	public function setUp() {
 		parent::setUp();
-		\OC_App::enable('files_antivirus');
```

Comment:
you can keep this now if required, because tests/ is now ignored by the code checker